app/editor/src/components/form/wysiwyg/CustomToolbar.tsx

```diff
@@ -14,8 +14,9 @@ export const CustomToolbar: React.FC<ICustomToolbarProps> = ({
   onClickRemoveFormat,
   onClickFormatRaw,
   onClickClear,
+  innerRef,
 }) => (
-  <div id="toolbar" className="toolbar">
+  <div ref={innerRef} className="toolbar">
```

Comment:
I'm no expert but this looks like it may be related to ForwardRef.  https://reactjs.org/docs/forwarding-refs.html
I don't fully understand it all, but you best read it over and make sure you're doing this correct.

Reply:
Sounds good, taking a look!